website_event_require_legal/models/event.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = "event.event"
+
+    website_require_legal = fields.Boolean(
+        string="Require legal terms",
+        help="If set, the user must have accepted the terms in event to register "
+        "attendees from the website.",
+    )
+    website_description_legal = fields.Html(
+        string="Legal Terms Description",
+        help="Custom text for the legal terms checkbox shown on the event "
```

Comment:
Put a placeholder saying that leaving it empty will put a default text

Reply:
All the changes have been made, just waiting for the tour to finish properly.

Reply:
I don't see the placeholder